packages/cgb-scripts/template/src/block/block.front.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * BLOCK: <% blockName %>
+ *
+ * Script to be executed in the editor AND on the frontend.
+ * 
+ */
+
+console.log( 'Hi! This executes in the editor and on the frontend' );
```

Comment:
What's the point of having this? Normal script are already doing that. Executing both on the backend and the frontend?

Reply:
The main block.build.js only executes in the editor b/c it's loaded with the `enqueue_block_editor_assets` action.

Am I wrong about this??